qp_shogun/shogun/tests/test_shogun.py

```diff
@@ -48,7 +49,8 @@ def setUp(self):
         self.params = {
             'Database': join(self.db_path, 'rep82'),
             'Aligner tool': 'bowtie2',
-            'Number of threads': 5
+            'Number of threads': 5,
+            'Percent ID': '0.95',
```

Comment:
Minor comment: this "ID" is "identity" not "identifier".

Reply:
that is correct, do you have a suggestion for the name or is it fine to leave as is?

Reply:
How about just "Percent identity"?